requirements.txt

```
@@ -5,7 +5,7 @@ SCons==4.8.1
 comtypes==1.4.6
 pyserial==3.5
 wxPython==4.2.2
-configobj @ git+https://github.com/DiffSK/configobj@e2ba4457c4651fa54f8d59d8dcdd3da950e956b8#egg=configobj
+configobj @ git+https://github.com/DiffSK/configobj@8be54629ee7c26acb5c865b74c76284e80f3aa31#egg=configobj
```

are we able to change this to a normal pip reference and use a stable release?

```suggestion
configobj==5.0.9
```